unstructured/file_utils/filetype.py

```diff
@@ -503,6 +503,7 @@ def document_to_element_list(
     include_page_breaks: bool = False,
     sort: bool = False,
     last_modification_date: Optional[str] = None,
+    **kwargs,
```

Comment:
Why do we need to pass down `**kwargs` here now?

Reply:
Because the `**kwargs` gets passed to `_add_element_metadata`, which is where the `epub_section` kwarg that's been passed down is added to the metadata